samplers/parser.go

```diff
@@ -72,9 +72,13 @@ func ParseMetricSSF(metric *ssf.SSFSample) (*UDPMetric, error) {
 		return nil, invalidMetricTypeError
 	}
 	h.Write([]byte(ret.Type))
-	ret.Value = float64(metric.Value)
+	if metric.Metric == ssf.SSFSample_SET {
+		ret.Value = metric.Message
+	} else {
+		ret.Value = float64(metric.Value)
+	}
 	ret.SampleRate = metric.SampleRate
-	tempTags := make([]string, len(metric.Tags))
+	tempTags := make([]string, 0)
```

Comment:
Do you mean my change fixed a bug, or that _is_ a bug? If it's a bug, I addressed it here: https://github.com/stripe/veneur/pull/207/commits/15e95d8b6b83b1e25c888e7bfac2ee34e247a856